servant/servant.cabal

```diff
@@ -60,6 +60,7 @@ library
     , http-types            >= 0.8  && < 0.10
     , mtl                   >= 2.0  && < 2.3
     , mmorph                >= 1    && < 1.1
+    , semigroups            >= 0.16 && < 0.19
```

Comment:
I'm worried about adding dependencies to `servant`, since this is what everything depends on.

Can we do this without `semigroups`? How many new transitive dependencies is this introducing?